### PR TITLE
Bump version 1.1.0

### LINKED ISSUE #53: There is a new release of the collection including the Conjur Ansible role
AC:
- [ ] README is reviewed for clarity about project state
- [ ] CHANGELOG is reviewed
- [ ] NOTICES is added / updated if needed

### PR DESCRIPTION
**Pull request by izgeri**
### What does this PR do?
- Does some minor clean up on tests
- Bumps version for 1.1.0 release that includes the Conjur Ansible role

### What ticket does this PR close?
Resolves #53 

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,11 +6,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+
 ### Added
 - The [Conjur Ansible role](https://galaxy.ansible.com/cyberark/conjur-host-identity) has been
   migrated to this collection, where it will be maintained moving forward.
+  At current, the role in the collection is aligned with the v0.3.2 release of
+  the standalone role.
   [cyberark/ansible-conjur-host-identity#30](https://github.com/cyberark/ansible-conjur-host-identity/issues/30)
-- Add `as_file` boolean option to store the secret as a temporary file and returns its path.
+- Add `as_file` boolean option to the lookup plugin which stores the secret as
+  a temporary file and return its path. This enables users to use the
```
Comment (diverdane) [inline CHANGELOG.md:18]: Should be "returns its path"?

File: README.md [hunk 2]
```diff
@@ -35,6 +35,17 @@ http://ecotrust-canada.github.io/markdown-toc/ -->
   v10.x+ accessible from the target node
 - Ansible >= 2.9
 
+## Using ansible-conjur-collection with Conjur OSS
+
+Are you using this project with [Conjur OSS](https://github.com/cyberark/conjur)? Then we
+**strongly** recommend choosing the version of this project to use from the latest [Conjur OSS
+suite release](https://docs.conjur.org/Latest/en/Content/Overview/Conjur-OSS-Suite-Overview.html).
```
Comment (diverdane) [inline README.md:42]: Would it be better to link directly to the Ansible-specific section of the Suite release docs? i.e.
https://docs.conjur.org/Latest/en/Content/Integrations/ansible.html

**Review by diverdane — APPROVED** [review]
LGTM.
Left one silly editorial nit comment, and one question about link to OSS suite docs.

File: README.md (hunk 2 above)
Comment (izgeri) [reply README.md:42]: in this case we are linking to the suite release stuff, not the ansible integration page. the version to use shows only in the suite release notes.

Comment (diverdane) [reply README.md:42]: Maybe the same link used below for `latest suite release` (https://docs.conjur.org/Latest/en/Content/ReleaseNotes/ConjurOSS-suite-RN.htm) would make it clearer what the latest suite release is? From the Release notes page, they can always navigate up to the overview.

**Review by diverdane — APPROVED** [review]
LGTM, left one more suggestion re. link to suite release page, but looks fine either way!

**Pull request merged**